### PR TITLE
use Aspect_DIR for cmake

### PR DESCRIPTION
**Pull request by gassmoeller**
This is a follow-up to #1050. I still occasionally got the problems described in there, and this change seems to fix the issue. Just so that I understand this correctly: We use the notation `ASPECT_DIR` for the shell environment variable and also inside the source-code when replacing input parameters, but the cmake build variable is called `Aspect_DIR`? So what should we output to screen, when setting up plugins? Currently we output `using ASPECT_DIR ...`.

### CODE REVIEW

File: benchmark/shear_bands/CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.8)
 
-FIND_PACKAGE(Aspect REQUIRED HINTS ${ASPECT_DIR} ../ ../../ $ENV{ASPECT_DIR})
+FIND_PACKAGE(Aspect REQUIRED HINTS ${Aspect_DIR} ../ ../../ $ENV{ASPECT_DIR})
```
Comment (bangerth) [inline benchmark/shear_bands/CMakeLists.txt]: Could we just use both `${Aspect_DIR}` and `${ASPECT_DIR}` as hints here? Same everywhere else.

**Comment by bangerth** [comment]
So is this all related to using `Aspect` as the package name, rather than `ASPECT`? (I think this was what #1050 was about, right @tjhei ?) We use the capitalized version everywhere else, so why don't we just use `ASPECT` as the package name instead?

File: benchmark/shear_bands/CMakeLists.txt (hunk 1 above)
Comment (gassmoeller) [inline benchmark/shear_bands/CMakeLists.txt]: That created cmake error messages for me. They were of the form: ... tried to access variable name ${ASPECT_DIR} that was not defined (I can look for the exact wording if you need it).

Comment (bangerth) [inline benchmark/shear_bands/CMakeLists.txt]: Isn't there a form of the kind `${value-or-empty-if-not-exist:ASPECT_DIR}`? I don't recall the exact syntax, but there should be something that allows using it even if not set.

**Comment by tjhei** [comment]
> So is this all related to using Aspect as the package name, rather than ASPECT? (I think this was what #1050 was about, right @tjhei ?)

Correct, our package name is "Aspect" and variables are case sensitive (at least with newer cmake versions). This is what #1050 fixed. I chose to keep the package name as "Aspect" otherwise every single external plugin would need to be updated. Our environment variable is ``ASPECT_DIR`` for historical reasons. 

This patch here is correct (fixed two places of two new plugins created by Juliane).

> so why don't we just use ASPECT as the package name instead?

We could change this if you want. This would be more in line with the fact that we call this software ASPECT too. I have a hard time finding the recommended capitalization for package names in cmake right now. I opened #1148.

**Pull request merged**